pkg/kv/kvserver/helpers_test.go

```diff
@@ -550,3 +550,16 @@ func WatchForDisappearingReplicas(t testing.TB, store *Store) {
 		}
 	}
 }
+
+// ChecksumRange returns a checksum over the KV data of the given range.
+func ChecksumRange(
+	ctx context.Context, desc roachpb.RangeDescriptor, snap storage.Reader,
+) ([]byte, error) {
+	var r *Replica // TODO(pavelkalinnikov): make this less ugly.
```

Comment:
`sha512()` doesn't use the `Replica` descriptor for anything, so we can just rename and export it instead.

Reply:
I'll see tomorrow if I can make this even cleaner, and will send a follow-up PR if so.